addons/stock/models/stock_picking.py

```diff
@@ -397,7 +397,7 @@ def _compute_move_line_exist(self):
 
     @api.one
     def _compute_has_packages(self):
-        self.has_packages = self.move_line_ids.filtered(lambda ml: ml.result_package_id)
+        self.has_packages = bool(self.env['stock.move.line'].search_count([('picking_id','=', self.id), ('result_package_id','!=',False)]))
```

Comment:
In fact, you don't need count all records for a boolean result:
I mean, you can use the first record found.
 - `self.env['stock.move.line'].search([('picking_id','=', self.id), ('result_package_id','!=',False)], limit=1)`

Notice the use of `search` instead of `search_count` and `limit=1`